show-land/sass/_typography.scss

```diff
@@ -0,0 +1,21 @@
+body {
+  font-family: "Lato", sans-serif;
+  font-weight: 400;
```

Comment:
Please use variables for font family and also font weight (you can get the font weight variable from shared folder) : )